### DISCUSSION TITLE
How to disable chromatic aberration 1.0.5a?

### DISCUSSION THREAD

**Discussion by vertexgamer** [sultim-t/xash-rt#129] [Q&A] [answered]
Is anyone aware on how to disable chromatic aberration and other post process effects? The in-game menù only allows to disable bloom, and i coulndn't find anything in the dev menù.

**Comment by vertexgamer** [comment]
an option do disable the denoiser would also be neat

**Reply by sultim-t** [reply]
Can be done in the "devmode" through a developer window: https://github.com/sultim-t/xash-rt/discussions/124

**Comment by sultim-t** [comment, marked as answer]
Just set `rt_ef_chraber` cvar to 0 in the valve/rt.cfg or in the console

**Reply by vertexgamer** [reply]
ok, thx
